.ci/pipeline/ci.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     displayName: 'apt-get'
   - script: |
       scripts/mklfpk.sh ${$(platform.type):3}
```

Comment:
The problem is that this construction doesn't work - ${$(platform.type):3} results in empty string.
Please fix it